api/src/main/java/org/openmrs/module/ohricore/engine/CommonsUUID.java

```diff
@@ -0,0 +1,15 @@
+package org.openmrs.module.ohricore.engine;
+
+/**
+ * @author MayanjaXL, Amos, Stephen, smallGod date: 30/06/2021
+ */
+public final class CommonsUUID {
+	
+	public static final String COMPUTED_CONCEPT_TARGET_ENCOUNTER = "UUID";
```

Comment:
@smallGod This should be a valid uuid for the Encounter Type

Reply:
I have deleted this for now, since I was not using it yet